### ISSUE TITLE
Environment modules: missing path entries

### ISSUE THREAD

**Issue by hansthen** [clustervision/trinity#235]
Module files should also search for module files in /cluster/modulefiles.

```
#/usr/share/Modules/your_contribs                       # Edit for your requirements
/trinity/clustervision/modulefiles
/cluster/modulefiles
```

**Comment by hansthen** [comment]
And this should be shared between login node and containers.

**Comment by abhimukh** [comment]
This is fixed in 284cdee83831ee9bcaeb6f6aeb279908e729c9d1

**Comment by themisa** [comment]
duplicate of #250
